src/shared/Internal.Shared.Tasks/build/Internal.Shared.Tasks.props

```diff
@@ -3,14 +3,14 @@
   <PropertyGroup>
     <_BuildToolsAssemblyTfm Condition="'$(MSBuildRuntimeType)' == 'Core'">netstandard1.6</_BuildToolsAssemblyTfm>
     <_BuildToolsAssemblyTfm Condition="'$(MSBuildRuntimeType)' != 'Core'">net46</_BuildToolsAssemblyTfm>
-    <_BuildToolsAssembly>$(MSBuildThisFileDirectory)..\tools\$(_BuildToolsAssemblyTfm)\Internal.AspNetCore.BuildTools.Tasks.dll</_BuildToolsAssembly>
+    <_BuildToolsAssembly>$(MSBuildThisFileDirectory)..\tools\$(_BuildToolsAssemblyTfm)\$(MSBUildThisFileName).dll</_BuildToolsAssembly>
```

Comment:
Why use `$(MSBuildThisFileName)` here? Looks like an unnecessary trick that'll break if we move to `Common.props` or similar.